python_client/MIQA.py

```diff
@@ -0,0 +1,130 @@
+from getpass import getpass
+import requests
+from s3_file_field_client import S3FileFieldClient
+
+
+from project import Project
```

Comment:
Ah, yes, we'll want to make the python client a proper package. Add an `__init__.py` to do so.

We should also consider making it a publishable package by moving the code down into its own directory and adding a `setup.py` so that it can be `pip` installable.